packages/mendel-pipeline/src/cache/entry.js

```diff
@@ -5,6 +5,7 @@ class Entry {
         // property is filled by cache where they have more context
         this.normalizedId;
         this.variation;
+        this.runtime; // one of 'broser' or 'server'
```

Comment:
I opened an issue for you to track constraints 😆 

Yeah... it is a bummer, but if we did all the huge job to support all combinations and edge cases, it would still be a good practice to organize your repo within the constraints. Makes things easier.